### PR TITLE
JP-2623: Fixing multiprocessing failure

### PR DESCRIPTION
**Pull request by kmacdonald-stsci**
The multiprocessing feature of ramp fitting was failing.  To ensure new features worked properly with multiprocessing the unneeded one group suppression list got removed and the `ZEROFRAME` handling got moved to after multiprocessing to ensure proper usage.

The primary cause of the multiprocessing failure was the `int_times` object, which is an `astropy.io` object that cannot be properly pickled.  Since it is not used in ramp fitting, it is no longer passed to `STCAL` ramp fitting.

### CODE REVIEW

**Review by hbushouse — APPROVED** [review]
Looks good to me. Needs a change log entry.

**Review by hbushouse — APPROVED** [review]

**Comment by hbushouse** [comment]
@kmacdonald-stsci can you fix the conflicts, which will then allow the CI tests to run again.

**Review by hbushouse — APPROVED** [review]
Looks good now.

**Comment by hbushouse** [comment]
@dmggh are you OK with this PR?

**Comment by jemorrison** [comment]
any way we can get this PR and the related one on JWST merged so we have multiprocessing available in master soonish ?

**Comment by hbushouse** [comment]
This also includes a minor, but noticeable, change to the way some cases of one good group are handled, which we were waiting for comment and testing from INS before merging. So we are not going to include this in our B8.0.1 release (which is being created today). Once the B8.0.1 release has been created we can then merge this anytime we want in order to get it into master.

**Comment by karllark** [comment]
I have used this branch and the accompanying `jwst` branch to process quite a large amount of MIRI imaging observations with success.  I have seen no differences in the resulting rate images.

**Comment by nden** [comment]
@PaulHuwe @ddavis-stsci Can one of you run a test with romancal to make sure the absence of int_times in the return values does not break romancal? It'd be good to merge this soon if possible.

**Comment by nden** [comment]
@kmacdonald-stsci Please assign also @ddavis-stsci and @PaulHuwe as reviewers of all stcal PRs to cover Roman testing.

**Review by ddavis-stsci — APPROVED** [review]
LGTM, the roman regression tests pass with Ken's changes, 
https://plwishmaster.stsci.edu:8081/me/my-views/view/all/job/RT/job/Roman-Developers-Pull-Requests/48/

**Review by PaulHuwe — APPROVED** [review]
All unit and regression tests passed in RomanCAL.

**Pull request merged**
